infra/templates/az-isolated-service-single-region/terraform.tfvars

```diff
@@ -0,0 +1,9 @@
+# GENERAL
+resource_group_location = "eastus2"
+name                    = "isolated-service"
+
+# SERVICE PLAN
+app_service_name = {
+  cobalt-backend-api-1 = "appsvcsample/static-site:latest"
```

Comment:
I'd imagine this definition of `app_service_name` would only be used for local testing. 

As part of our provisioning process, we should build a fresh image for each app service, similar to our single region template. The approach we took with the single region template was referencing the `https://github.com/erikschlegel/echo-server.git` image as a placeholder. The advantage is that we can tag each image to a release that's somewhat specific to the workspace ie `msftcse/cobalt-backend-api-1:release-devint`, and avoid creating a webhook on an image that's tagged to `latest`. 

```
app_service_name = {
    cobalt-backend-api = "msftcse/az-service-single-region:release" 
}
# ACR image deploy source
acr_build_git_source_url = "https://github.com/erikschlegel/echo-server.git"
```

The idea here is that app devs provide the image repo name and tag per app service, and we create a placeholder image. 

In the azure devops pipeline, `app_service_name` and `acr_build_git_source_url` will be provided as pipeline variables.

Reply:
See note about resources from the other subscription being out of scope. Based on this I think that we may want to put this off until ACR integration

Reply:
Based on our convo this will be done when we implement ACR integration for the template as outlined in https://github.com/microsoft/cobalt/issues/196